charms/kfp-api/src/charm.py

```diff
@@ -301,28 +297,6 @@ def _check_status(self):
                 raise ErrorWithStatus("Workload failed health check", MaintenanceStatus)
             self.model.unit.status = ActiveStatus()
 
-    def _upload_files_to_container(self, config_json):
-        """Upload required files to container."""
-        try:
-            self._check_container_connection(self.container)
```

Comment:
We should also remove this function if we don't need it anymore

Reply:
Agreed.